lib/lowkiq/queue/queue_metrics.rb

```diff
@@ -64,11 +64,9 @@ def coerce(result)
 
       def coerce_lag(res)
         _id, score = res.first
-
-        return 0 if score.nil?
-        return 1 if score == 0 # на случай Actions#perform_all_jobs_now
```

Comment:
is it intentional removal or by mistake?

Reply:
Intentional. I think it should be written in the shard_metrics file. But if the score variable is even zero, then the lag will be OK.